topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java

```diff
@@ -138,15 +136,18 @@ public void createMockKube(VertxTestContext context) throws Exception {
     }
 
     @AfterEach
-    public void tearDown() {
+    public void tearDown(VertxTestContext context) {
+        Checkpoint checkpoint = context.checkpoint(1);
```

Comment:
Not even a suggested change, but `context.checkpoint()` calls `context.checkpoint(1)` and generally in the rest of the code we have been calling the argument-less call. But that aside this looks like a good change to me.